src/components/Article.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+import { useParams } from 'react-router-dom';
+import axios from "axios";
+
+export default function Article()
+{
+    const { id } = useParams();
+    const [article, setArticle] = useState({});
+
+    const date = new Date(article.created_at);
+
+    useEffect(() => {
+        const fetchArticles = async () => {
+            const response = await axios.get(`https://nc-news-egcd.onrender.com/api/articles/${id}`);
```

Comment:
This can be refactored to use a baseURL with axios.create. Have a look at the notes as this will be a cleaner way to do things :)